dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -97,6 +97,18 @@ inline void ParseSampleDesc(std::vector<SampleDesc>& samples_container,
                IndexFileErrMsg(index_path, line, "no extensions provided for the sample"));
 }
 
+inline int ParseIndexVersion(const string& version_str) {
+  const char *s = version_str.c_str();
+  assert(*s == 'v');
```

Comment:
Assert or ENFORCE as we can read externally provided data?

Reply:
This is a private function that's invoked after we've checked that the version is among the supported ones. In that case, it would mean that the set of supported versions contains an invalid one.

Reply:
Ok